ladybug_vtk/to_vtk.py

```diff
@@ -0,0 +1,40 @@
+"""Functions to create VTK objects from Ladybug Objects."""
+
+import vtk
+from typing import List
+from ladybug_geometry.geometry3d import Point3D, Polyline3D
+from .polydata import PolyData
+
+
+def create_points(points: List[Point3D]) -> PolyData:
+    """Create Polydata from a Polyline."""
```

Comment:
This is a bit confusing. Shouldn't this method only return a PolyData with points? Or is this the one to create a polyline?